### ISSUE TITLE
monitoring stuck in df -k

### ISSUE THREAD

**Issue by bsislow** [dalibo/temboard-agent#381]
we have two clusters running on a given server denoted by two different ports.

we used the auto configure process to deploy both agents.

temboard metadata shows:

```
temboard=# select host_id from monitoring.hosts where hostname='myhost';
host_id
---------
      31
(1 row)

temboard=# select instance_id,port from monitoring.instances where host_id=31;
instance_id | port
-------------+-------
          31 | 45221
          32 | 45222
(2 rows)

temboard=# select count(*) from monitoring.metric_db_size_current where instance_id in (31,32);
count
-------
     0
(1 row)
```

the graphs for these agents are blank on the main temboard dashboard. status check graphs are also blank.

the following is repeating in the temboard agent log:

```
2020-01-15 11:33:02,234 temboardagent[14249]: [httpd] INFO: client: pem02.chi.accertify.net request: "GET /dashboard?key=ba2e1c48a5090d190af59c4f465f511a HTTP/1.1" 200 -
2020-01-15 11:33:04,201 temboardagent[14249]: [httpd] INFO: client: pem02.chi.accertify.net request: "GET /dashboard?key=ba2e1c48a5090d190af59c4f465f511a HTTP/1.1" 200 -
2020-01-15 11:33:06,199 temboardagent[14249]: [httpd] INFO: client: pem02.chi.accertify.net request: "GET /dashboard?key=ba2e1c48a5090d190af59c4f465f511a HTTP/1.1" 200 -
2020-01-15 11:33:08,189 temboardagent[14249]: [httpd] INFO: client: pem02.chi.accertify.net request: "GET /dashboard?key=ba2e1c48a5090d190af59c4f465f511a HTTP/1.1" 200 -
```

how can we fix these issues? what other data do you need? thanks.

**Comment by bersace** [comment]
@bsislow can you check monitoring configuration of agents ? And check for collector messages in agent's logs ?

**Comment by bersace** [comment]
Can you show what `systemctl status temboard-agent@*` shows ?

Monitoring configuration of agent is in `[monitoring]` section. The `collector_url` define the URL to the UI. It may be wrong.

**Comment by bsislow** [comment]
Can I email you the output?

**Comment by bersace** [comment]
@bsislow yes.

**Comment by bsislow** [comment]
Sorry can you send your email address? I do not have it.

**Comment by bersace** [comment]
@bsislow etienne D0T bersac AT dalibo D0T com . It's in every git commit ;-)

**Comment by bsislow** [comment]
This was our issue; resolved. Thanks for your help and patience.

**Comment by bersace** [comment]
temboard should help debugging such behaviour with a timeout and also, we should consider whether `--local` is suitable for temboard. Reopening.

**Comment by bersace** [comment]
Done. Will be in 7.9.

**Comment by bsislow** [comment]
FYI - we're on version 7.6 and will be upgrading to 7.9 this week.

This morning, our backup NFS mount crashed (`df -h` would hang, etc.) - this led to temboard halting monitoring on _all_ of our targets. I assume monitoring collection couldn't complete because of the hang when trying to access filesystems.

Will the fixes in 7.9 address this - and how do we need to adjust settings to do this (if needed)?

Thanks.

**Comment by bersace** [comment]
@bsislow yes, 7.9 will fix NFS mount crash handling by temBoard.
